client/src/App.js

```diff
@@ -130,8 +131,8 @@ const App = props => {
         className={classes.container}
       >
         <Grid item>
-          <Navbar theme={props.theme} {...props} />
-          <Navcrumbs {...props} />
+          <Navbar theme={props.theme}  />
```

Comment:
I'm getting an error when I try to log in or log out from this branch. Maybe it's because we're not spreading the props in here? On logout, I get `TypeError: Cannot read property 'logout' of undefined`. On login, I get `TypeError: Cannot read property 'show' of undefined`. They're both coming from Navbar.js. Might just be something on my end.

Reply:
Ahhh I think I forgot to declare lock={lock} thanks for that!